docs/alerts/creating-logs-alerts.md

```diff
@@ -1,41 +1,30 @@
 title: Creating Logs Alerts
 description: Step-by-step alert creation instructions for Logs
 
-In a Logs App, *saved queries* are used to save search queries that you want to reuse.
+In a Logs App you can create an Alert Rule which will trigger notifications when a condition is met.
 
 For example, let's say you used *include/exclude filters* to search for *Elasticsearch slowlogs warnings*.
 
 ![Elasticsearch slowlogs warnings search](../images/alerts/image_0.png)
 
-Clicking on the bell icon creates a new *saved query*, where filters are transformed into `Query string`.
+Clicking on the bell icon creates a new Alert Rule with query and all filters applied.
```

Comment:
Clicking on the bell icon creates a new Alert Rule with query and all filters applied.
=>
Clicking on the bell icon creates a new Alert Rule with the applied query and filters.